nomad/mock/alloc.go

```diff
@@ -87,14 +87,19 @@ func Alloc() *structs.Allocation {
 }
 
 func MinAlloc() *structs.Allocation {
-	job := MinJob()
+	return MinAllocForJob(MinJob())
+}
+
+func MinAllocForJob(job *structs.Job) *structs.Allocation {
```

Comment:
I could have sworn I split this out to a separate PR, like I did several other things, but I guess not.  think I should?

Reply:
Pretty trivial. Fine to leave in.